jpa-entity-generator-maven-plugin/src/main/java/com/smartnews/jpa_entity_generator/maven/EntityGenMojo.java

```diff
@@ -0,0 +1,28 @@
+package com.smartnews.jpa_entity_generator.maven;
+
+import com.smartnews.jpa_entity_generator.CodeGenerator;
+import com.smartnews.jpa_entity_generator.config.CodeGeneratorConfig;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+
+/**
+ * @goal generateAll
+ * @phase process-sources
+ */
+public class EntityGenMojo extends AbstractMojo {
```

Comment:
if we move this class to another place, we can remove it from `src/main/java` in the root directory.